### PR TITLE
Input Fields / Game Logic

### PR DESCRIPTION
**Pull request by flohip**
I added ALOT of stuff and should've split up the logic of the game into multiple user stories...
It's alot of plain Javascript , mixed with some React logic to render letters and color-in / disable buttons

### CODE REVIEW

File: components/Button/StartGame.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  return wonGame ? (
+    <>
+      <StyledHeader>
+        Du hast das Wort erraten!!! Hier sind ein paar Gummibärchen als
+        Belohnung...
+      </StyledHeader>
+      <Image
+        src={gummiBearPicture}
+        alt="a picture of gummi bears"
+        width="200"
```
Comment (heinenandreas) [inline components/Button/StartGame.js:18]: I´m not sure, does this work without a specification like px?

File: components/Keyboard/Key.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      {name}
+    </StyledButtons>
+  );
+}
+
+const StyledButtons = styled.button`
+  @media (min-width: 600px) {
+    width: 45px;
+    border-radius: 10px;
+
```
Comment (heinenandreas) [inline components/Keyboard/Key.js:15]: you might want to remove that empty line

File: components/Keyboard/Key.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    margin: 4px;
+  }
+  @media (max-width: 600px) {
+    width: 27px;
+    border-radius: 6px;
+
```
Comment (heinenandreas) [inline components/Keyboard/Key.js:21]: same here

File: utils/checkGuess.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    correctArray.push(guess);
+    return correctArray;
+  } else if (keyStateWrong === "wrong") {
+    const wrongArray = [];
+    wrongArray.push(returnValue);
+    wrongArray.push(keyStateWrong);
+    wrongArray.push(guess);
+    return wrongArray;
+  } else {
+    console.error("wtf happend here? ...");
```
Comment (heinenandreas) [inline utils/checkGuess.js:32]: Thats what I thought :D

**Review by heinenandreas — COMMENTED** [review]
Damn, nice work!

File: components/Button/StartGame.js (hunk 1 above)
Comment (flohip) [reply components/Button/StartGame.js:18]: Yep, the next/image width and height is by default set as px :)

**Pull request merged**